udata/settings.py

```diff
@@ -37,6 +37,7 @@ class Defaults(object):
 
     SECURITY_PASSWORD_HASH = b'bcrypt'
     SECURITY_PASSWORD_SALT = b'udata'
+    SECURITY_REGISTERABLE = True
```

Comment:
This is a flask-security variable; setting it to True lets new users register. Should I re-document the flask-security variables in doc/configuration.rst, or maybe just include a link to the flask-security documentation?

FWIW, googling "SECURITY_REGISTERABLE" finds the flask-security documentation.